src/instana/collector/utils.py

```diff
@@ -3,16 +3,17 @@
 from typing import TYPE_CHECKING, List
 
 from opentelemetry.trace.span import format_span_id
+from opentelemetry.trace import SpanKind
 
 if TYPE_CHECKING:
     from instana.span.span import InstanaSpan
 
 
-def format_trace_and_span_ids(
+def format_span(
     queued_spans: List["InstanaSpan"],
 ) -> List["InstanaSpan"]:
```

Comment:
This is not correct anymore. As what we have in the `collector.queued_spans` are `RegisteredSpan` or `SDKSpan` objects (see [here](https://github.com/instana/python-sensor/blob/a1329947a336047e4e04de378f81d625ada3eaa3/src/instana/recorder.py#L88)), we can change this to really reflect that. 

So, we can do like this:
```suggestion
def format_span(
    queued_spans: List[Union[RegisteredSpan, SDKSpan]],
) -> List[Union[RegisteredSpan, SDKSpan]]:
```
or this:
```suggestion
def format_span(
    queued_spans: List[Type["BaseSpan"]],
) -> List[Type["BaseSpan"]]:
```